airflow/www/forms.py

```diff
@@ -220,9 +220,9 @@ class ConnectionForm(DynamicForm):
             ),
         )
         description = StringField(lazy_gettext("Description"), widget=BS3TextAreaFieldWidget())
-        host = StringField(lazy_gettext("Host"), widget=BS3TextFieldWidget())
+        host = StringField(lazy_gettext("Host"), widget=BS3TextFieldWidget(), filters=[strip_filter])
         schema = StringField(lazy_gettext("Schema"), widget=BS3TextFieldWidget())
-        login = StringField(lazy_gettext("Login"), widget=BS3TextFieldWidget())
+        login = StringField(lazy_gettext("Login"), widget=BS3TextFieldWidget(), filters=[strip_filter])
```

Comment:
I get the fact that most systems may not allow hosts to contain leading or trailing whitespaces, but thinking critically and loud here in the case of login, would some users like to have intentionally leading or trailing whitespaces? I am not sure, just wanted to discuss this once :)

Reply:
I dont think so, it is not a common practice to have trailing or leading spaces. In between the login name is usually accepted. 

In my opinion, we can perform a strip and then if need be try to support leading/trailing if needed. WDYT?

Reply:
Well, I am not sure if stripping by default for all the fields would cause issues for someone's connections or not. If we're sure that it would not cause any issues then it is fine :)

Reply:
It shouldn’t, since no spaces are technically allowed in any of the text fields anyway (except descriptions). It would be a bug elsewhere if a connection requires a space to work in any of those fields.